packages/langium-cli/langium-config-schema.json

```diff
@@ -40,6 +40,50 @@
                         "out"
                     ]
                 },
+                "parserConfig": {
+                    "description": "An object to describe the parser configuration",
+                    "type": "object",
+                    "properties": {
+                        "chevrotainConfig": {
+                            "description": "An object to describe the Chevrotain parser configuration",
+                            "type": "object",
+                            "properties": {
+                                "recoveryEnabled": {
+                                    "description": "Is the error recovery / fault tolerance of the Chevrotain Parser enabled",
+                                    "type": "boolean"
+                                },
+                                "maxLookahead": {
+                                    "description": "Maximum number of tokens the parser will use to choose between alternatives",
+                                    "type": "number"
+                                },
+                                "dynamicTokensEnabled": {
+                                    "description": "A flag to support Dynamically defined Tokens",
+                                    "type": "boolean"
+                                },
+                                "nodeLocationTracking": {
+                                    "description": "Enable computation of CST nodes location",
+                                    "type": "string"
+                                },
+                                "errorMessageProvider": {
+                                    "description": "A custom error message provider",
+                                    "type": "object"
```

Comment:
I guess this config option cannot be set via JSON, so we should rather remove it from here.